packages/netlify-cms-backend-github/src/API.js

```diff
@@ -24,9 +24,9 @@ export default class API {
     this.useOpenAuthoring = config.useOpenAuthoring;
     this.repo = config.repo || '';
     this.originRepo = config.originRepo || this.repo;
-    this.repoURL = `/repos/${this.repo}`;
+    this.repoURL = this.repo ? `/repos/${this.repo}` : '';
     // when not in 'useOpenAuthoring' mode originRepoURL === repoURL
-    this.originRepoURL = `/repos/${this.originRepo}`;
+    this.originRepoURL = this.originRepo ? `/repos/${this.originRepo}` : '';
```

Comment:
`repoURL` will always be an empty string for `git-gateway` as it is overwritten here: https://github.com/netlify/netlify-cms/blob/c1c95874f41bd2b435c39d3dea263c77eaa601eb/packages/netlify-cms-backend-git-gateway/src/GitHubAPI.js#L10

I think we should add the same for `originRepoURL` (I missed it in this fix https://github.com/netlify/netlify-cms/pull/2631)

Reply:
I think that's why it was partially working. `repoURL` was ok but `originRepoURL` not